management/sites/sites.php

```diff
@@ -1,8 +1,8 @@
-<div class="wdn-inner-wrapper wdn-inner-padding-no-top">
-	<div class="wdn-grid-set">
+<div class="dcf-wrapper dcf-pb-8">
+	<div class="dcf-grid">
 		<!-- Site select list -->
-		<div class="wdn-col-one-fourth">
-			<ul class="unstyled-list wdn-list-reset">
+		<div class="dcf-col-25%-start dcf-pr-10">
+			<ul class="unstyled-list dcf-list-bare">
```

Comment:
Can the `unstyled-list` class and the associated css for it be removed now that we have `dcf-list-bare`?